### ISSUE TITLE
Running screenshots fails on CI

### ISSUE THREAD

**Issue by nickmeinhold** [nickmeinhold/crowdleague#99]
```sh
Changing locale from en-US to en-AU on 'test'...

error: closed

Unhandled exception:
command failed: exitcode=1, cmd='/Users/runner/Library/Android/sdk/platform-tools/adb -s emulator-5554 shell setprop persist.sys.locale en-AU ; setprop ctl.restart zygote', workingDir=null
#0      cmd (package:screenshots/src/utils.dart:427:5)
#1      changeAndroidLocale (package:screenshots/src/run.dart:544:3)
#2      setEmulatorLocale (package:screenshots/src/run.dart:522:5)
#3      Screenshots.runTestsOnAll (package:screenshots/src/run.dart:290:19)
<asynchronous suspension>
#4      Screenshots.run (package:screenshots/src/run.dart:127:11)
<asynchronous suspension>
#5      screenshots.<anonymous closure> (package:screenshots/src/run.dart:48:26)
#6      AppContext.run.<anonymous closure> (package:tool_base/src/base/context.dart:154:29)
#7      _rootRun (dart:async/zone.dart:1190:13)
#8      _CustomZone.run (dart:async/zone.dart:1093:19)
#9      _runZoned (dart:async/zone.dart:1630:10)
#10     runZoned (dart:async/zone.dart:1550:10)
#11     AppContext.run (package:tool_base/src/base/context.dart:153:18)
#12     runInContext (package:screenshots/src/context_runner.dart:17:24)
#13     screenshots (package:screenshots/src/run.dart:47:12)
#14     main (file:///Users/runner/hostedtoolcache/flutter/1.19.0-1.0.pre-dev/x64/.pub-cache/git/screenshots-d542991cf0d6fa10897746695f0aa4ed8bb973b4/bin/main.dart:122:25)
<asynchronous suspension>
#15     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#16     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
pub finished with exit code 255
```

**Comment by nickmeinhold** [comment]
I tried using the branch as described here: 
[Cannot start the IOS simulator · Issue #169 · mmcc007/screenshots](https://github.com/mmcc007/screenshots/issues/169)

Got the same result.

**Comment by nickmeinhold** [comment]
Fixed by changing the locale in [Fix screenshots · nickmeinhold/crowdleague@71c4039](https://github.com/nickmeinhold/crowdleague/commit/71c4039f3a3826279a9fd2df586fcbbabae0ded3)

Note: this caused a different problem - uploading to the play store now needs en-US Title etc. 
- tracking in #102
